model/AbstractRdsResultStorage.php

```diff
@@ -0,0 +1,727 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2014 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\taoOutcomeRds\model;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Exception\UniqueConstraintViolationException;
+use Doctrine\DBAL\Query\QueryBuilder;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Schema\SchemaException;
+use Doctrine\DBAL\Schema\Table;
+use oat\generis\persistence\PersistenceManager;
+use oat\oatbox\service\ConfigurableService;
+use oat\taoResultServer\models\classes\ResultDeliveryExecutionDelete;
+use oat\taoResultServer\models\classes\ResultManagement;
+use oat\taoResultServer\models\Exceptions\DuplicateVariableException;
+use taoResultServer_models_classes_Variable as Variable;
+
+/**
+ * Implements tao results storage using the configured persistence "taoOutcomeRds"
+ */
+abstract class AbstractRdsResultStorage extends ConfigurableService
+    implements \taoResultServer_models_classes_WritableResultStorage, \taoResultServer_models_classes_ReadableResultStorage, ResultManagement
+{
+    use ResultDeliveryExecutionDelete;
+    const SERVICE_ID = 'taoOutcomeRds/RdsResultStorage';
+    /**
+     * Constants for the database creation and data access
+     */
+    const RESULTS_TABLENAME = "results_storage";
```

Comment:
single quotes?  just to be consistent

Reply:
Sure. Just wanted to keep the changes to the bare minimum, to help for the PR reviewing process... Since it's old code, applying code style would go for almost every line changed and then review more difficult...